code/game/gamemodes/traitor/traitor.dm

```diff
@@ -18,6 +18,9 @@
 	var/traitors_possible = 4
 	/// How much the amount of players get divided by to determine the number of traitors.
 	var/const/traitor_scaling_coeff = 5
+	/// How much the traitor_scaling_coeff is multiplied by the gamemode as the above is a config global and I am not touching that
+	var/traitor_scaling_coeff_adjustment = 1
+
```

Comment:
is this needed? its never changed and its always set to 1

Reply:
I forgot to remove that from the original incarnation of trifecta being based off of traitor code. Will remove that